server/graphql/v2/object/AccountStats.js

```diff
@@ -149,6 +162,20 @@ export const AccountStats = new GraphQLObjectType({
           }
         },
       },
+      totalNetAmountReceived: {
+        description: 'Total net amount received',
+        type: new GraphQLNonNull(Amount),
+        async resolve(collective) {
+          const value = await collective.getTotalNetAmountReceived();
+          return { value, currency: collective.currency };
+        },
+      },
+      activeRecurringContributions: {
+        type: GraphQLJSON,
```

Comment:
I missed that part, we should have a proper type for this. `GraphQLJSON` is too broad